src/main/io/beeper.c

```diff
@@ -214,6 +223,8 @@ typedef struct beeperTableEntry_s {
     { BEEPER_ENTRY(BEEPER_BLACKBOX_ERASE,        18, beep_2shortBeeps,     "BLACKBOX_ERASE") },
     { BEEPER_ENTRY(BEEPER_ALL,                   19, NULL,                 "ALL") },
     { BEEPER_ENTRY(BEEPER_PREFERENCE,            20, NULL,                 "PREFERRED") },
+    { BEEPER_ENTRY(BEEPER_CAM_CONNECTION_OPEN,   20, beep_camConnectionOpen,                 "CAM_CONNECTION_OPEN") },
```

Comment:
Align text CAM_CONNECTION_OPEN and CAM_CONNECTION_CLOSE, ... with the other elements of the list. Maybe you will ned to add a space to all the other elements.